Sources/LocalConsole/LCManager.swift

```diff
@@ -477,6 +477,69 @@ public class LCManager: NSObject, UIGestureRecognizerDelegate {
         }
     }
     
+    public var isCharacterLimitDisabled = false
+    public var isCharacterLimitWarningDisabled = false
+    
+    public var showDebugMenuItem = true {
+        didSet { menuButton.menu = makeMenu() }
+    }
+    public var showUserDefaultsMenuItem = true {
+        didSet { menuButton.menu = makeMenu() }
+    }
+    public var showViewFramesMenuItem = true {
+        didSet { menuButton.menu = makeMenu() }
+    }
+    public var showSystemReportMenuItem = true {
+        didSet { menuButton.menu = makeMenu() }
+    }
+    public var showDisplayReportMenuItem = true {
+        didSet { menuButton.menu = makeMenu() }
+    }
+    public var showRestartSpringboardMenuItem = true {
+        didSet { menuButton.menu = makeMenu() }
+    }
+    
+    // Displays all UserDefaults keys, including unneeded keys that are included by default.
+    public var showAllUserDefaultsKeys = false
+    
+    /// Print items to the console view.
+    public func print(_ items: Any) {
+        let _currentText: String = {
+            if currentText == "" {
+                return "\(items)"
+            } else {
+                return currentText + "\n\(items)"
+            }
+        }()
+        
+        // Cut down string if it exceeds 50,000 characters to keep text view running smoothly.
+        if _currentText.count > 50000 && !isCharacterLimitDisabled {
+            
+            if !hasShortened && !isCharacterLimitWarningDisabled {
+                hasShortened = true
+                Swift.print("LocalConsole's content has exceeded 50,000 characters.\nTo maintain performance, LCManager cuts down the beginning of the printed content. To disable this behaviour, set LCManager.shared.isCharacterLimitDisabled to true.\nTo disable this warning, set LCManager.shared.isCharacterLimitWarningDisabled = true.")
```

Comment:
Hey @duraidabdul, sorry I didn't add this code, this is existing code I moved up to be within the Public marker rather than Private since it's a public func.